core/modules/user.py

```diff
@@ -320,12 +309,11 @@ def pwrecover(self, *args, **kwargs):
             # Update the password, save the user, reset his session and show the success-template
             user_skel["password"] = skel["password"]
             user_skel.toDB()
-            session["user.auth_userpassword.pwrecover"] = None
 
             return self.userModule.render.view(None, tpl=self.passwordRecoverySuccessTemplate)
 
     @tasks.CallDeferred
-    def sendUserPasswordRecoveryCode(self, userName: str, recoveryKey: str) -> None:
+    def sendUserPasswordRecoveryCode(self, user: UserSkel, recovery_key: str, user_agent: dict) -> None:
```

Comment:
Can you call this with a `UserSkel` deferred? I think this cannot be serialized.

Reply:
Yes, it will run locally, but not in the cloud. Provide the user's key and load the skeleton.